helm/dagster/schema/schema/charts/dagster/subschema/daemon.py

```diff
@@ -81,6 +81,7 @@ class Daemon(BaseModel):
     startupProbe: kubernetes.StartupProbe
     annotations: kubernetes.Annotations
     runMonitoring: Dict[str, Any]
+    runRetries: Dict[str, Any]
```

Comment:
Could make this into a typed object - just need to assert it against the config type. See how we did it with the compute logs manager for example: https://sourcegraph.com/github.com/dagster-io/dagster/-/blob/helm/dagster/schema/schema_tests/test_instance.py?L508%3A93=

Reply:
Ah nice, will follow up